src/protos/io/defang/v1/fabric.proto

```diff
@@ -325,6 +325,7 @@ message TailRequest {
   repeated string services = 1;
   google.protobuf.Timestamp since = 2;
   string etag = 3;
+  bool build = 4;
```

Comment:
I think we should make this an enum. Allows us to expand to other types of filter.

Reply:
I agree. gRPC design guidelines actually discourage bools in messages, so an enum makes sense. First member (0) is always UNSPECIFIED_SOMETHING and then 1 could be BUILD etc.. Let's not add types we don't actually have yet (ie. crash)

Reply:
I'd also often wish we had a simple text "filter", akin to `defang tail | grep asdf`, so perhaps a good time to add that too? No regex, just plain.

Reply:
> I'd also often wish we had a simple text "filter", akin to defang tail | grep asdf, so perhaps a good time to add that too? No regex, just plain.

Should we go ahead and call it `pattern`? We don't need to support regex right away.